model/src/main/kotlin/VcsType.kt

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Copyright (C) 2019 Bosch Software Innovations GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonValue
+
+/**
+ * A class for Version Control System types. Each type has one or more [aliases] associated to it, where the first
+ * alias is the definite name.
+ */
+data class VcsType(val aliases: List<String>) {
+    /**
+     * A constructor that searches [all known VCS aliases][ALL_ALIASES] for the given [type] and creates an instance
+     * with matching aliases, or an instance with only [type] as the alias if no aliases match.
+     */
+    @JsonCreator
+    constructor(type: String): this(
+        ALL_ALIASES.find { aliases ->
+            aliases.find { alias ->
```

Comment:
nit: replacing the inner find with 'any' would be a bit shorter.

Reply:
Good point, done.